lib/payloadAction.ts

```diff
@@ -21,6 +22,11 @@ export const  handlePayloadActions = async (app: IApp, read: IRead,  modify: IMo
             const targetDepartment: string = await getAppSettingValue(read, AppSetting.FallbackTargetDepartment);
             if (actionName) {
                 if (actionName === ActionIds.PERFORM_HANDOVER) {
+                    if (!targetDepartment) {
```

Comment:
@Shailesh351 Remind me please.....  are there conditions where we get ActionIds.PERFORM_HANDOVER that are not payloads that came from Dialogflow.agent or our Fulfillment server?

Reply:
@ear-dev Perform handover is happening in below cases,

1) Payloads that coming from Dialogflow.agent or our Fulfillment server
2) Quick Reply button configured for handover
3) Incoming Endpoint
4) if fallbackThreshold reached on dialogflow and is configured in the settings
5) Failure on calling Dialogflow detectIntent API

Reply:
please remove this block.

Reply:
Done 👍